src/Frontend.hs

```diff
@@ -45,14 +48,24 @@ import qualified Persistent.Implementation.STM
 ----------------------------------------------------------------------
 -- driver
 
-runFrontend :: Config.Config -> IO ()
+extendClearanceOnSessionToken :: Applicative m => ThentosSessionToken -> m ()
+extendClearanceOnSessionToken _ = pure () -- FIXME
```

Comment:
Probably this a consequence of generalizing thentos just enough to be used elsewhere. It is not clear that this is the right hook for aula.